api/controller.js

```diff
@@ -15,20 +12,14 @@ var controllers = {
        }
        res.json(aboutInfo);
    },
+   
    get_distance: function(req, res) {
```

Comment:
Same with this - is distance used for anything?

Reply:
Good catch.  It probably would help to calculate the distance between user and event, after we figure out how to get events' location. And the distance can help to do the sorting.